wagtail/wagtailcore/tests/test_blocks.py

```diff
@@ -1670,6 +1670,32 @@ class ShoppingListBlock(blocks.StructBlock):
         )
         self.assertIn('value="chocolate"', form_html)
 
+    def render_form_with_classname(self, classname):
+        class LinkBlock(blocks.StructBlock):
+            title = blocks.CharBlock()
+            link = blocks.URLBlock()
+
+        block = blocks.ListBlock(LinkBlock, classname=classname)
```

Comment:
Shouldn't the argument be called `form_classname` just like the corresponding attribute in `StructBlock`'s Meta class? 

Actually, it seems that the naming is not entirely consistent, as FieldPanels have a `classname` argument that affects their rendering in the admin, whereas for StructBlock the argument is called `form_classname`.

Reply:
@mnieber  - thanks for the comment, gonna attempt to pick this one back up.

Yes, it appears that Blocks use meta.classname but StructBlocks use meta.form_classname.

I will update this with `form_classname` and re-submit (applied to the latest code).